src/react-components/src/components/02-molecules/Cards/EditionCard.tsx

```diff
@@ -12,14 +12,9 @@ export type EditionDetails = {
   language: string | JSX.Element,
   license: string | JSX.Element,
 
-  /** readOnlineLink and downloadLink take in either urls as a string,
```

Comment:
Moved these comments from the `type` to the interface props so the component's documentation generator can pick it up. I think it also makes more sense to have the comments there since those define the props better and not the type.